src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/LicenceService.kt

```diff
@@ -56,4 +63,72 @@ class LicenceService(
       addressTown = addressObject.addressTown,
       postCode = addressObject.postCode,
     )
+
+  private fun formatCurfewHoursObject(curfewHours: CurfewHours, licenceId: Long?): List<CurfewTimes> {
+    val day1 =
+      CurfewTimes(
+        licenceId = licenceId,
+        fromDay = "Monday",
```

Comment:
There's a [Day of week](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/DayOfWeek.html) enum we should really be using for this.

That goes for in CVL also - better to use a type to represent a list of known values rather than raw strings. Luckily java has one built in for this domain of data